### ISSUE TITLE
more meaningful default parameters for training and add minimal example to README.md

### ISSUE THREAD

**Issue by philippmuench** [hiddengenome/deepG#99]
best would be that `trainNetwork(path= "example_files/fasta", use.cudnn = F, run.name  ="my_first_model")` would would work on a normal laptop.

- default path.val should be path when no extra validation set is supplied
- keep network small, layer.size = 128, batch.size = 64, layers.lstm = 2, steps.per.epoch = 512,  epochs = 5
- on default setting, when no tensorboard.log is given, there should nothing be written except the checkpoints
 
for minimal example best would be
- add one or multiple E.coli genome to example_files/fasta folder, not sure if the current content is meaningful, you may want to name that folder example_files/train
- highlight that use.cuddnn should be enabled on a GPU machine. 
- add also a minimal example on how to use this model to start the shiny app 
- add also a minimal example how to produce hdf5 file based on that model and a input sequence
